Doc/library/tracemalloc.rst

```diff
@@ -330,6 +331,14 @@ Functions
    :mod:`tracemalloc` module as a tuple: ``(current: int, peak: int)``.
 
 
+.. function:: get_traced_refs()
+
+   Get the current count of created and destroyed traced references.
+   :mod:`tracemalloc` module as a tuple: ``(created: int, destroyed: int)``.
+
```

Comment:
You should explain what are "references" here. They are not the classical reference count. Maybe explain the difference with the reference count?